data/src/test/java/com/onboarding/data/RoomMapperTest.kt

```diff
@@ -0,0 +1,103 @@
+package com.onboarding.data
+
+import com.onboarding.data.databse.entity.RoomCharacter
+import com.onboarding.data.databse.entity.RoomLocationLink
+import com.onboarding.data.mapper.transform
+import com.onboarding.data.mapper.transformToRoom
+import com.onboarding.data.util.ConstantUtil.INT_VALUE
```

Comment:
The constants used in the tests should be declared in the test. 
Do not use the ConstantUtil class, define the constants in the companion pls

Reply:
done